### PR TITLE
Solved issue #1413 

### PR DESCRIPTION
**Pull request by Ngai-E**
1.changed the email input type from 'text' to 'email'.
2.validated filled email entry input to check fo validity using javaScript regex pattern match before saving.
3.displayed messages to users with iziToast

### CODE REVIEW

**Comment by Ngai-E** [comment]
@muarachmann  I have added iziToast and removed the trailing spaces please check it out.

File: templates/pharmacies/general_edit.php [hunk 1]
```diff
@@ -83,10 +84,29 @@
...
+                document.pharmacy.submit();
+                //Z&H Removed redirection
+            }
+            else{
+                <?php
+                    $msg = 'Please enter a valid email address';
+                    echo ("var alertMsg ="."'".htmlspecialchars( xl($msg), ENT_QUOTES)."'".";\n");
+                ?>
+                    iziToast.warning({
+                        position : "bottomRight",
```
Comment (muarachmann) [inline templates/pharmacies/general_edit.php]: perfect @Ngai-E its cool

Comment (muarachmann) [reply templates/pharmacies/general_edit.php]: Testing this ...

**Comment by Ngai-E** [comment]
@muarachmann thanks for the review waiting for the merge to complete my report. For now can I just go ahead and be working on other opened issues and send PR?

**Comment by muarachmann** [comment]
Yes open more issues, send more PRs. I will ask Art to review your PR so it
gets merged.

Happy coding
Cheers

**Comment by aethelwulffe** [comment]
Note: put the actual context in your pull request name.  That is what shows up in the history.  Makes more sense when we are looking at history or blame in the repo.

**Pull request merged**